pkg/driver/node.go

```diff
@@ -127,10 +129,39 @@ func (d *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not create volume: %s, %v", volumeID, err)
 	}
+
 	if err := jfs.BindTarget(ctx, bindSource, target); err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not bind %q at %q: %v", bindSource, target, err)
 	}
 
+	if cap, exist := volCtx["capacity"]; exist {
+		capacity, err := strconv.ParseInt(cap, 10, 64)
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "invalid capacity %s: %v", cap, err)
+		}
+		quotaPath := volCtx["subPath"]
+		var subdir string
+		for _, o := range mountOptions {
+			pair := strings.Split(o, "=")
+			if len(pair) != 2 {
+				continue
+			}
+			if pair[0] == "subdir" {
+				subdir = path.Join("/", pair[1])
+			}
+		}
+		settings, err := d.juicefs.Settings(ctx, volumeID, secrets, volCtx, options)
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "get settings: %v", err)
+		}
+		output, err := d.juicefs.SetQuota(ctx, secrets, settings, path.Join(subdir, quotaPath), capacity)
```

Comment:
get `subPath` from settings and `subdir` from settings.Options, it's trimSpaced.

Reply:
Done